operations/security_and_authentication/Change_NCN_Image_Root_Password_and_SSH_Keys.md

```diff
@@ -332,7 +332,7 @@ This step updates the entries in BSS for the NCNs to use the new images.
                 kernel=$(cat bss_$xname.json | jq '.[]  .kernel')
                 initrd=$(cat bss_$xname.json | jq '.[]  .initrd')
                 params=$(cat bss_$xname.json | jq '.[]  .params')
-                cray bss bootparameters update --initrd $initrd --kernel $kernel --params $params --name $xname --format json
+                cray bss bootparameters update --initrd $initrd --kernel $kernel --params $params --host $xname --format json
```

Comment:
I think this is wrong and the option should be `--hosts`

Reply:
updates my type-o